IndicoToolkit/fileprocessing.cs

```diff
@@ -0,0 +1,69 @@
+using System;
+using System.IO;
+using System.Linq;
+using System.Collections.Generic;
+
+namespace IndicoToolkit
+{
+    public class FileProcessing
+    {
+        public List<string> FilePaths; 
+        public List<string> DefaultAcceptedTypes; 
+        
+        public FileProcessing ( List<string> filePathsInput = null)
+        {
+            FilePaths = filePathsInput ?? new List<string>();
+            DefaultAcceptedTypes = new List<string>() {"*.pdf", "*.doc", "*.docx", "*.tiff", "*.tif", "*.png"};
```

Comment:
should probably just set this on line 11. if we aren't allowing them to pass in accepted types here, no reason to use the constructor to set it.